### ISSUE TITLE
Formatted cell work with editable extension

### ISSUE THREAD

**Issue by AaricChen** [wenzhixin/bootstrap-table#2072]
for instance

``` javascript
{ isPlayer : true }
```

And I want to convert display name in table shown as "Yes" or "No" for "true" and "false".

when I work with formatter and editable, the display name always "Empty".
How to fix this ?

**Comment by AaricChen** [comment]
I have also see #1819 , but It seem not work yet.

**Comment by MauricioTRS** [comment]
I changed these lines and it worked:

```javascript
column.formatter = function(value, row, index) {
    if(typeof column._formatter == "string") {
        column._formatter = window[column._formatter];
    }
    var result = column._formatter ? column._formatter(value, row, index) : value;
```

**Comment by wenzhixin** [comment]
Fixed in #1819
